### PR TITLE
update python config to mention env vars

### PR DESCRIPTION
**Pull request by danilito19**
Update the python configuration page to explain how to use environment variables.

### CODE REVIEW

File: content/installation/python/PythonConfig.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ tags: "installation python django flask pyramid agent service configuration"
 -->
 
 The Python agent and Contrast Service use a YAML file to alter the agent behavior. Go to the Python [YAML Template](installation-pythonconfig.html#python-template) for fully formatted properties that you can copy and use in your own configuration files.
+Environment variables may also be used to override any configuration in the YAML file.
```
Comment (kmckeedocs) [inline content/installation/python/PythonConfig.md:8]: Please write in active voice:

You can also use environment variables to override any configuration...

File: content/installation/python/PythonConfig.md [hunk 2]
```diff
@@ -18,6 +19,15 @@ The configuration file is named *contrast_security.yaml* or *contrast_security.y
 
 The agent and service may share a common configuration file, but only some options and sections are applicable to each process.
 
+
+## Environment Variables
+
+In addition to using the YAML file, you can use environment variables to specify every configuration option supported by the 
+YAML file. Environment variable names are derived from the YAML path by
+replacing path segment delimiters (`.`) with double underscores (`__`). For example, `agent.logger.level` would be defined as
+`CONTRAST__AGENT__LOGGER__LEVEL` while `agent.service.logger.path` as `CONTRAST__AGENT__SERVICE__LOGGER__PATH`.
```
Comment (kmckeedocs) [inline content/installation/python/PythonConfig.md:28]: For consistency, replace "as" with "would be"

File: content/installation/python/PythonConfig.md [hunk 3]
```diff
@@ -18,6 +19,15 @@ The configuration file is named *contrast_security.yaml* or *contrast_security.y
...
 The agent and service may share a common configuration file, but only some options and sections are applicable to each process.
 
+
+## Environment Variables
+
+In addition to using the YAML file, you can use environment variables to specify every configuration option supported by the 
+YAML file. Environment variable names are derived from the YAML path by
+replacing path segment delimiters (`.`) with double underscores (`__`). For example, `agent.logger.level` would be defined as
+`CONTRAST__AGENT__LOGGER__LEVEL` while `agent.service.logger.path` as `CONTRAST__AGENT__SERVICE__LOGGER__PATH`. 
+The Python Agent will give precedence to any environment variable over its corresponding value defined in the YAML file.
```
Comment (kmckeedocs) [inline content/installation/python/PythonConfig.md:29]: Python agent (lowercase a)

**Comment by danilito19** [comment]
@kmckeedocs thank you for the very helpful feedback! It's ready for final review.

**Pull request merged**